docs/adr/0002-use-slf4j-for-logging.md

```diff
@@ -0,0 +1,31 @@
+# Use slf4j together with log4j2 for logging
+
+* Currently JabRef uses apache commons logging 1.2 for logging errors and messages. However, this is not compatible with java 9 and is superseeded by log4j.
+* SLF4J provides a facade for several logging frameworks, including log4j and supports already java 9
+* Log4j is already defined as dependency and slf4j has already been required by a third party dependency
+
+## Considered Alternatives
+
+* [Log4j2](https://logging.apache.org/log4j/2.x/)
+* [SLF4J](https://www.slf4j.org/)
```

Comment:
I think, the options are:

- Log4J2
- SLF4J with Logback binding
- SLF4J with Log4J binding

Because of https://logging.apache.org/log4j/2.0/log4j-slf4j-impl/, it seemed, you chose the Log4J binding